From dc004a26b97d0595aa2f02cf8a6ce78dcbcc92d0 Mon Sep 17 00:00:00 2001
From: BoNic <BoNio9527@gmail.com>
Date: Thu, 30 Nov 2023 21:19:37 -0500
Subject: [PATCH] slim

---
 config/packages.mk | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/config/packages.mk b/config/packages.mk
index a93d8dc..fd5d68d 100644
--- a/config/packages.mk
+++ b/config/packages.mk
@@ -19,17 +19,10 @@ PRODUCT_PACKAGES += \
     Launcher3QuickStep \
     GameSpace \
     OmniJaws \
-    Papers \
     RepainterServicePriv \
     ThemePicker \
     Updates
 
-TARGET_BUILD_VIMUSIC ?= false
-ifeq ($(TARGET_BUILD_VIMUSIC),true)
-PRODUCT_PACKAGES += \
-    ViMusic
-endif
-
 # Whether the compiled build to be shipped with Aperture CAM
 TARGET_BUILD_APERTURE_CAMERA ?= true
 ifeq ($(TARGET_BUILD_APERTURE_CAMERA),true)
-- 
2.25.1

