From 218e810ee3fc258a6bf1d0b973d870a877dff65e Mon Sep 17 00:00:00 2001
From: BoNic <BoNio9527@gmail.com>
Date: Sat, 25 Nov 2023 08:26:23 -0500
Subject: [PATCH] HOME-deserves-to-wake-up-devices-just-as-well-as-bac

---
 core/java/android/view/KeyEvent.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/java/android/view/KeyEvent.java b/core/java/android/view/KeyEvent.java
index 9c015979c..7c688015a 100644
--- a/core/java/android/view/KeyEvent.java
+++ b/core/java/android/view/KeyEvent.java
@@ -2068,6 +2068,7 @@ public class KeyEvent extends InputEvent implements Parcelable {
             case KeyEvent.KEYCODE_CAMERA:
             case KeyEvent.KEYCODE_FOCUS:
             case KeyEvent.KEYCODE_MENU:
+            case KeyEvent.KEYCODE_HOME:
             case KeyEvent.KEYCODE_PAIRING:
             case KeyEvent.KEYCODE_STEM_1:
             case KeyEvent.KEYCODE_STEM_2:
-- 
2.25.1

